alvr/dashboard/src/data_sources.rs

```diff
@@ -18,7 +22,7 @@ use tungstenite::{
     http::{HeaderValue, Uri},
 };
 
-const REQUEST_TIMEOUT: Duration = Duration::from_millis(200);
+const REQUEST_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
I don't think we should be changing the global request timeout to something that high.
Also if the popup appears after two seconds it can be quite disruptive ig, there's like a sweet window where the application has launched but ur probably not yet interacting, that's probably more like a second. Otoh if ur in a region with really high latency ig it's needed to function and  for other people it doesn't happen? Tho I think 1 second is probably the sweetspot in general, it's best effort anyway.

Reply:
But also, i don't think a popup after 2 seconds would be that distracting tbh, that's not enough time to do anything really

Reply:
Yea sure leave it as 2s, but separate out the timeouts.